.circleci/config.yml

```diff
@@ -15,16 +15,21 @@ references:
 
 jobs:
   test:
-    working_directory: /go/src/github.com/fairwindsops/insights-cli
+    working_directory: /home/circleci/go/src/github.com/fairwindsops/insights-cli
     docker:
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.19
     steps:
       - checkout
       - run:
           name: lint
           command: |
-            test -z $(go fmt ./...)
-            go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
+            fmt_output=$(go fmt ./...)
```

Comment:
The shell changed or was updated in the new CircleCI Go image, and `test -z` returns an error when `go fat` returns more than one line.